app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
I am not sure. My intuition is that it is not necessary.

The job IDs are UUIDs, users are going to really need to guess. The UUIDs are also ephemeral, so I don't think that students would be able to guess them. Furthermore, the files are actually available of S3 even without any key...

So I'm not sure what kind of extra security we're adding here.